src/main/java/FlightList.groovy

```diff
@@ -0,0 +1,57 @@
+import java.time.LocalDateTime
+
+interface FlightList {
+    List<Flight> getAllFlightsTo(Airports destination)
+
+    List<Flight> getAllFlightsFrom(Airports from)
+
+    void printFlights()
+
+    void printFlightsTo(Airports destination)
+
+    void printFlightsFrom(Airports to)
+
+    int getPassengerCount()
+
+    int getPassengerCountTo(Airports to)
+
+    int passengerAmountFrom(Airports from)
```

Comment:
all these should be named in the same way. "calculatexxxx" since it uses a list of flights.